File: 02703.py
for i in range(int(input())):
    message = input()
    secret = input()

    result = ""
    for j in range(len(message)):
        if message[j].isalpha():
            result += secret[ord(message[j]) - ord('A')]
        else: result += message[j]

    print(result)

File: 02703.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
	int T; cin >> T;
	getchar();

	for (int i = 0; i < T; i++) {
		string message; getline(cin, message);
		string secret; getline(cin, secret);

		for (int j = 0; j < message.length(); j++) {
			if (isalpha(message[j])) {
				message[j] = secret[message[j] - 'A'];
			}
		}

		cout << message << endl;
	}
}
